cluster/config-defaults.yaml

```
@@ -404,6 +404,11 @@ skipper_open_policy_agent_decision_logs_buffer_type_event_enable: "false"
 # Decision logging sets the maximum number of decision log events that can be buffered before being dropped
 skipper_open_policy_agent_decision_logs_buffer_type_event_limit: "10000"
 
+# Open Policy Agent JWT cache configuration
+skipper_open_policy_agent_jwt_cache_enable: false
```

```suggestion
skipper_open_policy_agent_jwt_cache_enable: "false"
```

All config-item values must be a string.